Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.12
```

Comment:
There is 3.12.1 with bugfix, should we use that instead?

Reply:
changelog from alpine: https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases

Reply:
alpine uses floating tags. So pulling 3.12 actually pulls 3.12.1.

You can see here that the hashes are the same. https://hub.docker.com/_/alpine?tab=tags

Reply:
TIL! Thanks.